mobileversion.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <meta charset="UTF-8" >
+     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="mobileversion.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+     <title>Mobile Version Website</title>
+  </head>
+    <body>
+       <div class="container">
+         <div class="item1">
+         <img src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl.png" alt="snapshoot">
+         <section class="ton">
+            <h1 class="tonic">Tonic</h1>
+         </section>
+         <section class="sec">
+            <p class="mainp">
+            <h5 class="canopy">CANOPY</h5>
+            <p class="mid">&#8226;</p>
+            <p class="back">Back End Dev</p>
+            <p class="mid">&#8226;</p>
+            <p class="year">2015</p>
+            </p>
```

Comment:
- [ OPTIONAL] **You can use here ul tag with li tag rather than a section tag**
```
<ul class="sec">
            <li class="canopy">CANOPY</li>
            <li class="mid">&#8226;</li>
            <li class="back">Back End Dev</li>
            <li class="mid">&#8226;</li>
            <li class="year">2015</li>
</ul>
```